index.html

```diff
@@ -44,8 +44,8 @@
                 <h3>Hello! Sharing World</h3>
                 <h1>Haitian Otaku,<br> Global Summit<br>2023</h1>
                 <p class="heroDesc"> Lorem ipsum, dolor sit amet consectetur adipisicing elit. Accusantium dolor a quod asperiores fugit iure facere ipsum cumque est? Eveniet perspiciatis molestias explicabo, aliquid aperiam temporibus quasi velit dolorum odio?</p>
-                <h2>2023.08.30(Wed) - 31(Thu) </h2>
-                <p>Fubar Club of Port-au-Prince, Art Center Nabi and more</p>
+                <h2>2023.08.30(Wed) ~ 31(Thu) </h2>
+                <p class ="adress">Fubar Club of Port-au-Prince, Art Center Nabi and more</p>
             </div>
         </section>
```

Comment:
Certainly, here's a revised review with your feedback in mind:

- [ ] 👍 Impressive progress has been made based on the previous reviewer's suggestions, yet a few more adjustments are needed for the mobile version:
    - [ ] As previously mentioned, ensure that the size of elements, margins, and paddings are adjusted to align with the Behance design. Elements should be left-aligned and started from the same position, with consistent padding throughout.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/25349a7c-136d-4573-8dd0-015a55004f03)| ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/51ec91a2-01df-4eb8-a51b-7584459f46c1)|

Continuing to fine-tune these details will bring your project even closer to the design's intended look and feel. Keep up the fantastic work! 🌟📝🚀